pymongo_auth_aws-1.1.0.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pymongo_auth_aws-1.1.0.dist-info/LICENSE,sha256=xx0jnfkXJvxRnG63LTGOxlggYnIysveWIZ6H3PNdCrQ,11357
pymongo_auth_aws-1.1.0.dist-info/METADATA,sha256=Lkd0q6feBEWDgUI5-rxiOveOLhCjqRAG8RiBm2oAOGo,4924
pymongo_auth_aws-1.1.0.dist-info/RECORD,,
pymongo_auth_aws-1.1.0.dist-info/WHEEL,sha256=z9j0xAa_JmUKMpmz72K0ZGALSM_n-wQVmGbleXx2VHg,110
pymongo_auth_aws-1.1.0.dist-info/top_level.txt,sha256=P21opOPUCOCYmeePHPABoeATMGhdoQqi0BeiIRVr348,17
pymongo_auth_aws/__init__.py,sha256=vVXmWNpo2AelXeZjPr10gU0FKzLB_w3iIS7RP4xki2A,751
pymongo_auth_aws/__pycache__/__init__.cpython-311.pyc,,
pymongo_auth_aws/__pycache__/auth.cpython-311.pyc,,
pymongo_auth_aws/__pycache__/errors.cpython-311.pyc,,
pymongo_auth_aws/__pycache__/version.cpython-311.pyc,,
pymongo_auth_aws/auth.py,sha256=1k7MLpzS_l5FLplNEwzPcr_Ietrp0K-PaK1lSkroR_8,8584
pymongo_auth_aws/errors.py,sha256=ilEemepO2UD-cW5C7ZjiQWhZCmidDHlSheoWB1_2rro,725
pymongo_auth_aws/version.py,sha256=SvdFEZflUGQBcnlxCKtyTZc34t172HfkiTf8zZ4FlkY,604
